.github/workflows/balena.yml

```
@@ -122,3 +133,5 @@ jobs:
               ((retry_count++))
             fi
           done
```

You'll need to fail the check if you reach here:

```suggestion
          done
          exit 1
```